google-cloud-bigtable/.rubocop.yml

```diff
@@ -2,7 +2,9 @@ AllCops:
   Exclude:
     - "google-cloud-bigtable.gemspec"
     - "lib/google/bigtable/v2/**/*"
```

Comment:
Might be worth it to change this line to `"lib/google/bigtable/**/*"`, as there are still Rubocop errors in `lib/google/bigtable/admin/v2/...`:

```ruby
     - "lib/google/bigtable/**/*"
```

Reply:
Whoops, missed this. Changed.